styles/main.css

```diff
@@ -107,3 +161,140 @@ header {
   background: none;
   cursor: pointer;
 }
+
+/* End Hero Section */
+
+/* Start Work Section */
+
+.work {
+  margin-inline: 2.4rem;
+  padding-bottom: 10rem;
+}
+
+.title--work {
+  text-transform: capitalize;
+  padding: 2rem;
+}
+
+.title--work::after,
+.title--about::after {
+  content: "";
+  display: block;
+  border-bottom: 4px solid var(--secondary-color);
+  border-radius: 2.4rem;
+  width: 4.8rem;
+  margin: 2.4rem auto;
+}
+
+.card-container {
+  display: -ms-grid;
+  display: grid;
+  grid-template-columns: repeat(1, 1fr);
+  gap: 3rem;
+}
+
+.card {
+  width: 33rem;
+  height: 47rem;
+  border: 1px solid #d0d9d4;
+  border-radius: 0.8rem;
+}
+
+.card__imgbox {
+  height: 60%;
+  background-color: var(--bg-card);
+}
+
+.card__img {
+  width: 100%;
+  height: 100%;
+  padding-block: 10%;
+}
+
+.card__details {
+  height: 40%;
+  display: -webkit-box;
+  display: flex;
+  -webkit-box-orient: vertical;
+  -webkit-box-direction: normal;
+  -ms-flex-direction: column;
+  flex-direction: column;
+  -ms-flex-pack: distribute;
+  justify-content: space-around;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  align-items: center;
+}
+
+.card__text {
+  font-size: 2.4rem;
+  font-weight: 800;
+  color: var(--text-color);
+  text-transform: capitalize;
+}
+
+.card__list,
+.about__card__list {
+  width: 100%;
+  display: -webkit-box;
+  display: flex;
+  justify-content: space-evenly;
+  flex-wrap: wrap;
+}
```

Comment:
- [ ] Compare your design on the right side and the template one on the left side then:
- Reduce the space between technologies(ruby on rails, bootstrap, HTML, CSS...) by changing the `justify-content: space-evenly` with `justify-content: space-center`

![Screenshot from 2021-11-26 03-57-17](https://user-images.githubusercontent.com/65068771/143515807-b6ebf063-f231-4a3c-a6c9-5388e04e465d.png)